### DISCUSSION TITLE
pyAEDT hfss.Reference_cs_to_global including a point

### DISCUSSION THREAD

**Discussion by AndrewKee** [ansys/pyaedt#3069] [Ideas]
Due to the nature of all points being in reference to the global coordinate system, it is very convenient to have the function "global_to_cs(point, ref_cs)"
There are however other uses for having the inverse function.  The function "reference_cs_to_global(cs)" exists, however one cannot pass a point to be translated as well.  
Rather than having to dig up the function for doing the quaternion and the translation, having this built in would be very useful.

In summary, can we add the ability to pass a point in the "reference_cs_to_global(cs)" function?

**Comment by Samuelopez-ansys** [comment]
Hi @AndrewKee ,

Please submit in Issues a new feature. If you know how to implement it, I suggest you to implement the feature, if you are external to Ansys, you just need to create a fork, and then we will review the Pull request. PyAEDT is an open source code, so everyone can contribute :) 

If not, we will prioritize the implementation from the Issues list.

Thanks!
